src/ipyvizzu/templates/ipyvizzu.js

```diff
@@ -1,187 +1,213 @@
+    });
+  }
 
-    log(element, chartId, chartProperty) {
-      if (IpyVizzu.nbconvert) IpyVizzu._hide(element);
-      this.charts[chartId] = this.charts[chartId].then((chart) => {
-        console.log(chart[chartProperty]);
-        return chart;
+  log(element, chartId, chartProperty) {
+    this.charts[chartId] = this.charts[chartId].then((chart) => {
+      console.log(chart[chartProperty]);
+      return chart;
+    });
+  }
+
+  control(element, method, prevId, lastId, ...params) {
+    this.controls[prevId].then(() => {
+      this.controls[lastId].activated.then((control) => {
+        if (method === "seek") {
+          const value = params[0];
+          control[method](value);
+          return;
+        }
+        if (method === "store") {
+          const id = params[0];
+          this.storage[id] = control[method]();
+          return;
+        }
+        control[method]();
       });
-    }
+    });
+  }
 
-    control(element, method, prevId, lastId, ...params) {
-      if (IpyVizzu.nbconvert) IpyVizzu._hide(element);
-      this.controls[prevId].then(() => {
-        this.controls[lastId].activated.then((control) => {
-          if (method === "seek") {
-            const value = params[0];
-            control[method](value);
-            return;
-          }
-          if (method === "store") {
-            const id = params[0];
-            this.storage[id] = control[method]();
-            return;
-          }
-          control[method]();
-        });
+  _moveHere(chartId, element) {
+    element.append(this.elements[chartId]);
+  }
+
+  _scroll(chartId, enabled) {
+    if (!IpyVizzu.inhibitScroll && enabled) {
+      this.elements[chartId].scrollIntoView({
+        behavior: "auto",
+        block: "center",
       });
     }
+  }
 
-    _moveHere(chartId, element) {
-      if (IpyVizzu.nbconvert)
-        IpyVizzu._display(this.elements[chartId], element);
-      element.append(this.elements[chartId]);
-    }
+  static _hide(element) {
+    document.getElementById(
+      element.selector.substring(1),
+    ).parentNode.style.display = "none";
+  }
 
-    _scroll(chartId, enabled) {
-      if (!IpyVizzu.inhibitScroll && enabled) {
-        this.elements[chartId].scrollIntoView({
-          behavior: "auto",
-          block: "center",
-        });
-      }
+  static _display(prevElement, element) {
+    if (prevElement.parentNode) {
+      prevElement.parentNode.style.display = "none";
     }
+    document.getElementById(
+      element.selector.substring(1),
+    ).parentNode.style.display = "flex";
+    document.getElementById(
+      element.selector.substring(1),
+    ).parentNode.style.margin = "auto";
+  }
 
-    static _hide(element) {
-      document.getElementById(
-        element.selector.substring(1),
-      ).parentNode.style.display = "none";
+  static changeAnalyticsTo(analytics) {
+    if (IpyVizzu.analytics !== analytics) {
+      console.log("ipyvizzu gather usage stats:", analytics);
```

Comment:
Is this debug message intentional?

Reply:
Yes, streamlit also display a msg like this.